### DISCUSSION TITLE
Taildwind styling not applying in my React + Vite app

### DISCUSSION THREAD

**Discussion by tanno1** [tailwindlabs/tailwindcss#12794] [Help]
Hey, I have been banging my head against a wall for awhile trying to figure this out and I cannot figure out why my tailwind is not applying in my react app. I am thinking it is something really simple like my folder organization and an improper link reference, but I don't know. 

In the image, you can see that there are class attributes that I would think would style it in the tailwind way, but i don't know why. No errors are thrown in console log?
![image](https://github.com/tailwindlabs/tailwindcss/assets/94481731/2eaad261-62af-4809-ae70-ccf3a64a6ebd)

The repo for this is here: https://github.com/tanno1/intro_proj_client
The only thing I can see that I did wrong was adjust the folder configuration of the config.js files but I really don't know. I am trying to work with react router v6 as well and I think in my eagerness to implement that, I complicated the tailwind stuff. I pushed the most recent stuff I have. Let me know if anyone has any ideas or if you see something wrong

**Comment by makibat** [comment]
Hey!

Looks like you forgot to import your `index.css` file in `main.jsx`

``` js
import React from 'react';
import { createRoot } from 'react-dom/client';
import App from './App';
import './index.css';

// Use createRoot for React 18
const root = createRoot(document.getElementById('root'));

root.render(
  <React.StrictMode>
    <App />
  </React.StrictMode>
);

```

Also move your postcss, tailwind and vite config to the root of your app and update the paths to your files in `tailwind.config.js`:
``` js 
content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}']
```

Hope that helps!

**Reply by tanno1** [reply]
Thanks @makibat for the quick response. That fixed it!
